### ISSUE TITLE
Workflow "Is Active" filter in Dataview

### ISSUE THREAD

**Issue by markewampler** [SparkDevNetwork/Rock#3410]
### Prerequisites
* [ x] Put an X between the brackets on this line if you have done all of the following:
  * Can you reproduce the problem on a fresh install or the [demo site]

### Description
A clear and concise description of what the bug is, include screenshots if applicable.

### Steps to Reproduce
Create a dataview based on a workflow
Add the filter for "Is Active"
save

**Expected behavior:**

See a list of Active Workflows

**Actual behavior:**

See Screenshot.
WARNING: There was a problem with one or more of the filters for this data view...

The specified type member 'IsActive' is not supported in LINQ to Entities. Only initializers, entity members, and entity navigation properties are supported
![screen shot 2018-11-08 at 10 28 15 am](https://user-images.githubusercontent.com/33845027/48212501-5730de80-e341-11e8-9dab-e76b62c5fc16.png)

### Versions
* **Rock Version:** [8.3]
* **Client Culture Setting:** [en-us]

┆Attachments: <a href="https://app.asana.com/app/asana/-/get_asset?asset_id=1113387516863638">image.png</a>

**Comment by edmistj** [comment]
Mark is right. IsActive is a virtual property with the following logic

```
get
{
    return ActivatedDateTime.HasValue && !CompletedDateTime.HasValue;
}
```

Probably need to get this to not show in dataviews.

**Comment by nairdo** [comment]
(FYI: We closed this because we believe it was corrected in v8.7)
